composer.json

```diff
@@ -23,12 +23,13 @@
     "sort-packages": true
   },
   "require": {
-    "php": "^5.5 || ^7.0",
+    "php": ">=5.5",
```

Comment:
This is fairly significant change on it's own and would be best broken out into a seperate request we can submit independent of this change. (This should hopefully be a quick one.)

Reply:
I restored it. I will create a sperate PR later.